index.js

```diff
@@ -25,12 +105,18 @@ function createTeamWindow(team) {
 			'partition': `persist:${team}`,
 			'plugins': false,
 
+			'preload': path.join(__dirname, 'browser.js'),
+
 			// fails without this because of CommonJS script detection
-			'node-integration': false
+			'node-integration': false,
+
+			// required for Hipchat page title updates
+			'web-security': false
```

Comment:
Yeah I was looking at it again now. It was needed for ping-stuff, until I got Notifications working. So the comment is wrong, but we need it off if we want to see the silly images we send to each other in the chat. From the docs:

> web-security Boolean - When setting false, it will disable the same-origin policy (Usually using testing websites by people), and set allow_displaying_insecure_content and allow_running_insecure_content to true if these two options are not set by user.